src/main/java/com/nike/cerberus/operation/core/UploadCertFilesOperation.java

```diff
@@ -43,12 +43,19 @@
  */
 public class UploadCertFilesOperation implements Operation<UploadCertFilesCommand> {
 
-    private static final String CA_FILE_NAME = "ca.pem";
-    private static final String CERT_FILE_NAME = "cert.pem";
-    private static final String KEY_FILE_NAME = "key.pem";
-    private static final String PKCS8_KEY_FILE_NAME = "pkcs8-key.pem";
-    private static final String PUB_KEY_FILE_NAME = "pubkey.pem";
-    public static final Set<String> EXPECTED_FILE_NAMES = ImmutableSet.of(CA_FILE_NAME, CERT_FILE_NAME, KEY_FILE_NAME, PKCS8_KEY_FILE_NAME, PUB_KEY_FILE_NAME);
+    private static final String CHAIN_FILE_NAME = CertificateService.CERT_CHAIN_FILE;
```

Comment:
Javadoc explaining each file type

Reply:
Why, I changed all the names to be self describing if thats not enough then an example of what you are thinking would be nice.

Reply:
E.g. "PKCS8_KEY_FILE_NAME" could have javadoc saying "Private key in PKCS8 format"

Reply:
E.g. "CHAIN_FILE_NAME" could have javadoc saying "Certificate Authority chain".  Just to make them more obvious when we add new developers.

Reply:
nvm, I will reference the constants from the cert service

Reply:
- [ ] Done